simtools/corsika/corsika_config.py

```diff
@@ -246,6 +247,9 @@ def _validate_and_convert_argument(self, par_name, par_info, value_args_in):
             value_args = [0 * par_info["unit"][0], value_args[0]]
         elif par_name == "PRMPAR":
             value_args = self._convert_primary_input_and_store_primary_name(value_args)
+        elif par_name == "ESLOPE":
+            # To make it easier to pass to sim_telarray
```

Comment:
I am not sure if this comment is helpful. It didn't tell me anything. Is this more a motivational comment why self.eslope was introduced? If yes, then it should be removed.

Reply:
Yes it is exactly that, a motivational comment cause otherwise it seems sort of out of place. I agree it should be removed though so I will remove it.